src/__mocks__/popper.js.js

```diff
@@ -0,0 +1,12 @@
+import PopperJs from 'popper.js';
+
+export default class Popper {
+    static placements = PopperJs.placements;
+
+    constructor() {
+        return {
+            destroy: () => {},
+            scheduleUpdate: () => {},
+        };
+    }
+}
```

Comment:
Is duplicated extension in `popper.js.js` made on purpose?

Reply:
yes. the package called `popper.js`. for Jest to see use it as a mock automatically each time it needs the package, we need to add `.js` after the package name. hence `popper.js.js`